tests/i18n/patterns/tests.py

```diff
@@ -463,3 +463,25 @@ def test_kwargs(self):
             tpl.render(Context({})).strip().split(),
             ["/vertaald/apo/", "/traduzidos/apo/"],
         )
+
+
+class URLExampleTests(URLTestCaseBase):
+    def test_demo(self):
```

Comment:
Test and test class names should describe shortly what we want to test. Phrases `demo` or `example` are not appropriate.

Reply:
Hi, So I've changed the names of both the Test and test class, I hope that it does the job.